pages/common/banner.md

```diff
@@ -0,0 +1,15 @@
+# banner
+
+> Print large ASCII art version of the given argument.
```

Comment:
Perhaps `Prints text as large ASCII art.` would read better?

Reply:
Thanks for your suggestion. My proposed description takes inspiration from the one of the _echo_ command. I would keep the root form of the verb, as in the majority of the other commands. But I like "Print the given argument as a large ASCII art."
What do you think?

Reply:
Yeah, go for `Print the given argument as a large ASCII art.` then :slightly_smiling_face: